matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/timeline/TimelineEvent.kt

```diff
@@ -53,6 +54,18 @@ data class TimelineEvent(
         }
     }
 
+    fun getDisambiguatedDisplayName(): String {
+        return if (isUniqueDisplayName) {
+            senderName ?: root.senderId
+        } else {
+            senderName?.let {
+                it + " (" + root.senderId + ")"
+            }
+                    ?: root.senderId
```

Comment:
Let the root.senderId be outside of the if as you write it twice.

Reply:
done